Recon/python/simpleTrigger.py

```diff
@@ -39,8 +39,8 @@ class TriggerProcessor(ldmxcfg.Producer) :
     def __init__(self,name) :
         super().__init__(name,'recon::TriggerProcessor','Recon')
 
-        self.beamEnergy = 4000.
-        self.thresholds = [ 1500.0, 1000. + self.beamEnergy, 500. + 2*self.beamEnergy, 100. + 3*self.beamEnergy ]  #toy something up 
+        self.beamEnergy = 8000.
```

Comment:
I was wondering if we should deduce this number based on the detector geometry name, and not do any hardcoded number? i.e. `ldmx-det-v14-8gev` will be 8000, otherwise 4000? I guess what it really should be is that if the det name contains the `8gev` string than 8000, otherwise 4000

Reply:
sure, if you want to code that up that would be very convenient for the user. since it's configurable and i assumed most trigger studies would target 8 GeV i haven't gone to any lengths to harmonize it, but it would for sure be safer to do it that way. if you have the time and energy?

Reply:
OK I did it in https://github.com/LDMX-Software/ldmx-sw/pull/1245/commits/cffc5ba7722c6042c06d6cc6f1bf7bd3396de7bf

Please note the ECAL pn validation config still uses the 4 GeV geom + beam, so updating the trigger to 8GeV would have lead to some inconsistencies (although it's just a test file, so maybe fine?)

Reply:
@bryngemark does this look ok for you?

Reply:
yeah originally #1229 should deal with this. i'll put in a catch-all issue for whatever we don't get to now.

Reply:
i hadn't realised your change would affect every existing use of the trigger processor. does it have to be made a mandatory argument? then why not just ship the beam energy?

Reply:
I think having it a mandatory argument is actually helpful, that will enforce the user to think about it, while if there is a default then that can lead to a situation where the beam energy is changed but the trigger settings were forgotten to be changed. Shipping the beam energy as a mandatory argument would lead to the same situation indeed. But then I just like the idea that we govern everything with the `det` string.

Reply:
ok, i suppose we can be forced to remember this :) 
but the more i think of it, the trigger decision is informed by beam energy more than it needs to know detector version, so i think this is the more logical thing to explicitly pass to it. tbh i thought you would implement reading the detector under-the-hood, from something else than an argument (not that i had a clear picture of how you'd achieve it). and i am probably biased by my assumption that in the long run, there will be no need to include the beam energy in the detector name, as we will only have 8 GeV beam. 

if your argument is that more people explicitly set the detector than the beam energy, then i might disagree but i'll fold, now you have my arguments, i trust your judgement.

Reply:
>  tbh i thought you would implement reading the detector under-the-hood,

yes, I tried to do that first but it seems the simple trigger class does not see that info directly

Reply:
Done in https://github.com/LDMX-Software/ldmx-sw/commit/c99cdcbf7bc342911c0d9239a6d5183a34c24d17